service/routes.py

```diff
@@ -86,7 +85,7 @@ def create_inventory_records():
         abort(status.HTTP_409_CONFLICT, error)
 
     inventory.create()
-    location_url = url_for("get_inventory_records", product_id=inventory.product_id, _external=True)
+    location_url = url_for("get_inventory_records", product_id=inventory.product_id, condition=inventory.condition, _external=True)
     statement = f"Inventory product with ID {inventory.product_id}"
     statement += f"and condition: {inventory.condition} created."
```

Comment:
can the statements 89-91 be made into one line for more readability

Reply:
The problem is that Pylint throws errors of more more characters in a line, that's why the two sentence is split in two separate lines

Reply:
you can use something like "\" to break a line into multiple lines. https://stackoverflow.com/a/53180
i feel this would still be a better approach